param/low_level_motion_sim.yaml

```diff
@@ -52,7 +52,7 @@ level_flight_required_height: 0.35
 level_flight_required_hysteresis: 0.10
 
 update_frequency: 60.0
-startup_timeout: 10.0
+startup_timeout: 20.0
```

Comment:
Is there a reason we changed this?

Reply:
This is just so that stuff doesn't time out in VMs, it shouldn't be in the PR

Reply:
So please revert this value back.

Reply:
Change is reverted